app/samples/udp_bcast/prj.conf

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024 RIT Launch Initiative
+# SPDX-License-Identifier: Apache-2.0
+
+# CPP
+CONFIG_CPP=y
+CONFIG_STD_CPP20=y
```

Comment:
more so a general question than about this file but are we saying c++20 project wide as our official version?

Reply:
Yes. If possible, I would like to stick with the latest versions of our dependencies if possible and there is little tradeoff in doing so. We can benefit from some of the newer features in C++20 if we choose to do so (i.e consteval)

Reply:
raaaaaaaaaa good answer